mage/installation.mdx

```diff
@@ -0,0 +1,119 @@
+---
+id: installation
+title: How to install MAGE?
+sidebar_label: Installation
+---
```

Comment:
I think this file needs to be removed, we already have `mage/installation.md `in the master branch.